roles/openshift_master/tasks/main.yml

```diff
@@ -114,12 +121,26 @@
 
 - name: Start and enable openshift-master
   service: name=openshift-master enabled=yes state=started
+  when: not openshift_master_ha
   register: start_result
 
 - name: pause to prevent service restart from interfering with bootstrapping
   pause: seconds=30
   when: start_result | changed
 
+- name: Install cluster packagese
+  yum: pkg=pcs state=present
+  when: openshift_master_ha and not openshift.master.cluster_defer_ha | bool
+  register: install_result
+
+- name: Start and enable cluster service
+  service: name=pcsd enabled=yes state=started
+  when: openshift_master_ha and not openshift.master.cluster_defer_ha | bool
+
+- name: Set the cluster user password
+  shell: echo {{ openshift_master_cluster_password | quote }} | passwd --stdin hacluster
```

Comment:
@twiest the user module was my first thought as well, and trying to find a standard way to create the password is where I kept getting stuck...  there was a Debian package that is recommended, but not available in Fedora/RHEL, there was an openssl command, but it would have required an additional package, and the other options would have required that I implement a more complex script.

Since this is a short term solution for HA anyway, I saw the shell module using passwd as the easiest method to maintain for the time being.